runtime-config-linux.md

```diff
@@ -156,11 +162,13 @@ The cgroups will be created if they don't exist.
    "cgroupsPath": "/myRuntime/myContainer"
 ```
 
-`cgroupsPath` can be used to either control the cgroups hierarchy for containers or to run a new process in an existing container.
+The container process will be added to the cgroups at `cgroupsPath`.
 
 You can configure a container's cgroups via the `resources` field of the Linux configuration.
 Do not specify `resources` unless limits have to be updated.
-For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
+For example, to run a new process in an existing container without updating limits, specify `cgroupsPath` but not `resources`.
+To run a new process in an existing container and update limits, specify both `cgroupsPath` and `resources`.
+To run a new process in a new container with explicit resource limits, specify `resources` and optionally set `cgroupsPath` to a cgroup that does not yet exist.
```

Comment:
On Wed, Dec 16, 2015 at 11:46:24AM -0800, Vish Kannan wrote:
> > -For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
> > +For example, to run a new process in an existing container without updating limits, specify `cgroupsPath` but not `resources`.
> > +To run a new process in an existing container and update limits, specify both `cgroupsPath` and `resources`.
> > +To run a new process in a new container with explicit resource limits, specify `resources` and optionally set `cgroupsPath` to a cgroup that does not yet exist.
> 
> Why can't a user run a new container using existing cgroups?

They can, but it wouldn't be a new container (and I tried to cover
that in the “in an existing container without updating limits” and “in
an existing container and update limits” cases, but can add
clarification about both of those requiring cgroupsPath pointing to an
existing cgroup.  Would that address your concerns?